Explorer/Assets/Plugins/TexturesFuse/TexturesServerWrap/Unzips/SemaphoreTexturesUnzip.cs

```diff
@@ -0,0 +1,33 @@
+using Cysharp.Threading.Tasks;
+using System;
+using System.Threading;
+using Utility.Types;
+
+namespace Plugins.TexturesFuse.TexturesServerWrap.Unzips
+{
+    public class SemaphoreTexturesUnzip : ITexturesUnzip
```

Comment:
Seems to be not required, right?

Reply:
First version, later it was replaced with pooled impl